File: Backend/Internal/Entities/Student.cs
namespace Hackaton_DW_2024.Internal.Entities;

public class Student
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public string StudentId { get; set; }
    public int GroupId { get; set; }
    public string? Telegram { get; set; }
    public string? PhoneNumber { get; set; }
}

File: frontend/src/entities/student/Student.ts
export interface Student{
    type: "student";
    groupId: number;
    id: number;
    name: string;
    patronymic?: string;
    phoneNumber: string | null;
    studentId: string;
    surname: string;
    telegram: string | null;
    userId: number;
}

export interface StudentDto{
    groupId: number;
    id: number;
    name: string;
    patronymic?: string;
    phoneNumber: string | null;
    studentId: string;
    surname: string;
    telegram: string | null;
    userId: number;
}